roofit/hs3/src/JSONIOUtils.cxx

```diff
@@ -0,0 +1,6 @@
+#include "JSONIOUtils.h"
```

Comment:
You have to register the `JSONIOUtils.cxx` also in `roofit/hs3/CMakeLists.txt` so that it gets built.

This is the reason why the CI failed.